src/mcfret.c

```diff
@@ -0,0 +1,366 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include <string.h>
+#include <time.h>
+#include <fftw3.h>
+#include "omp.h"
+#include "types.h"
+#include "NISE_subs.h"
+#include "mcfret.h"
+#include "project.h"
+
+
+
+
+   
+void mcfret(t_non *non){
+  /* Call the MCFRET Routine */
+   if (!strcmp(non->technique, "MCFRET") || (!strcmp(non->technique, "MCFRET-Autodetect")) || (!strcmp(non->technique, "MCFRET-Absorption"))
+        || (!strcmp(non->technique, "MCFRET-Emission")) || (!strcmp(non->technique, "MCFRET-Coupling")) || (!strcmp(non->technique,
+        "MCFRET-Rate")) || (!strcmp(non->technique, "MCFRET-Analyse")) ) {
+        printf("Perfroming the MCFRET calculation.\n");
+    }
+
+/* Define variables and arrays */
+   /* Integers */
```

Comment:
The description of the variables can be improved. "Integers" is already seen from the variable declaration. The descriptions of the floats is much better.